os-autoinst-obs-auto-submit

```diff
@@ -182,8 +182,14 @@ if [[ -e job_post_skip_submission ]]; then
 fi
 
 # throttle number of submissions to allow only one SR within a certain number of days
-if [[ $throttle_days != 0 ]] && $osc request list --project "$dst_project" --type submit --state new,review --mine --days "$throttle_days" | grep --quiet 'Created by'; then
-    echo "Skipping submission, there is still a pending SR younger than $throttle_days."
+if [[ $throttle_days != 0 ]] && $osc request list --project "$dst_project" --type submit --state new,review --mine --days "$throttle_days" \
+    | {
+        grep --quiet 'Created by'
+        rc=$?
+        cat > /dev/null
+        exit $rc
+    }; then
```

Comment:
I only understand why and how this works because you explained me in Slack. Might be worth a comment or something. `#black magic` might be fitting :laughing: